src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.IncrementalMemberEditAnalyzer.cs

```diff
@@ -83,9 +83,7 @@ public async Task<ImmutableDictionary<DiagnosticAnalyzer, ImmutableArray<Diagnos
                         // Check if we have existing cached diagnostics for this analyzer whose version matches the
                         // old document version. If so, we can perform span based incremental analysis for the changed member.
                         // Otherwise, we have to perform entire document analysis.
-                        var state = analyzerWithState.State;
-                        var existingData = analyzerWithState.ExistingData;
-                        if (oldDocumentVersion == existingData.Version)
+                        if (oldDocumentVersion == VersionStamp.Default)
```

Comment:
`existingDdata` would always be DocumentAnalysisData.Empty.  As such, this check is equivalent to checking if this has a default version stamp.